src/main/java/uk/gov/justice/hmpps/prison/service/HealthService.java

```diff
@@ -69,7 +68,7 @@ public PersonalCareNeeds getPersonalCareNeeds(final Long bookingId, final List<S
         return new PersonalCareNeeds(returnList);
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```

Comment:
Removed SYSTEM_USER as this is never actually picked up via the annotation - added test (disabled) to be addressed at a later date